pkg/configuration/configuration.go

```diff
@@ -43,6 +45,8 @@ type Configuration interface {
 	GetStringSlice(key string) []string
 	GetBool(key string) bool
 	GetBoolWithError(key string) (bool, error)
+	GetDuration(key string) time.Duration
```

Comment:
nitpicking: I'd create a new interface to remain backwards compatible. Something like `CacheableConfiguration`. I'd also rename the interface func to `GetCacheDuration`. `GetDuration` feels to generic to me.

Reply:
This function is a generic getter and has nothing specific todo with Cache. It is like all the other getters, it gives you out of the box support for accessing duration type values.